### PR TITLE
PAN-OS EDL Management - add edl_get_external_file_metadata_command

### PR DESCRIPTION
**Pull request by bakatzir**
fix bug when updating external file to non existent list

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18293
fixes: https://github.com/demisto/etc/issues/18294

## Screenshots
in related issues

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - https://github.com/demisto/etc/issues/18323
- [x] Code Review

### CODE REVIEW

File: Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py [hunk 1]
```diff
@@ -261,19 +261,24 @@ def edl_update_from_external_file(list_name: str, file_path: str, type_: str):
...
+        if type_ == 'merge':
+            unified = set_internal.union(set_external)
+            list_data_new = list(unified)
+        else:  # type_ == 'override'
+            list_data_new = list(set_external)
+        dict_of_lists.update({list_name: list_data_new})
+        demisto.setIntegrationContext(dict_of_lists)
+        return list_data_new
+    else:
+        demisto.log(str(file_data))
```
Comment (yuvalbenshalom) [inline Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py]: what is the size we are expecting to log here?

Comment (bakatzir) [reply Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py]: i should remove this line.

Comment (yuvalbenshalom) [reply Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py]: @bakatzir take a look on the next line with the log

**Comment by bakatzir** [comment]
@kirbles19 
yml:
https://github.com/demisto/content/blob/267524c60f0c46b8e0e8fc5682a18c3881843f61/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.yml
md:
https://github.com/demisto/content/blob/267524c60f0c46b8e0e8fc5682a18c3881843f61/Releases/LatestRelease/PaloAltoNetworks_PAN_OS_EDL_Management.md

**Comment by kirbles19** [comment]
@bakatzir .yml and .md updated, all done.

**Pull request merged**